nx/lib/nx.ex

```diff
@@ -5245,6 +5245,38 @@ defmodule Nx do
     impl!(tensor).conjugate(%{tensor | type: Nx.Type.to_complex(tensor.type)}, tensor)
   end
 
+  @doc """
+  Calculates the complex phase angle of each element in the tensor.
+  If $z = a + bi = r e^\\theta$, $phase(z) = atan2(b, a)$
+
+  ## Examples
```

Comment:
```suggestion
  ### Examples

```

Reply:
I undid this too as the other functions under Nx use level 2 headers